ereuse_devicehub/resources/action/views.py

```diff
@@ -32,7 +32,7 @@ def save_json(req_json, tmp_snapshots, user):
     month = now.month
     day = now.day
     hour = now.hour
-    minutes = now.min
+    minutes = now.minute
 
     name_file = f"{year}-{month}-{day}-{hour}-{minutes}_{user}_{uuid}.json"
```

Comment:
It would be possible to follow this pattern **yyyy-mm-dd-hh:mm** like appears on #74 , to better distinguish the date from the hour.

`name_file = f"{year}-{month}-{day}-{hour}:{minutes}_{user}_{uuid}.json"`

@cayop I'm sorry I didn't see it before, it's a very minor change, could it be done directly to testing?